src/glue/jobs/json_to_parquet.py

```diff
@@ -582,12 +585,11 @@ def add_index_to_table(
         else:
             selectable_original_field_name = original_field_name
         logger.info(f"Adding index to {original_field_name}")
-        parent_index = parent_table.select(
-            (
-                [selectable_original_field_name, "cohort"]
-                + INDEX_FIELD_MAP[table_data_type]
-            )
-        ).distinct()
+        index_fields = INDEX_FIELD_MAP[table_data_type]
```

Comment:
Given that now `Participant_Identifier` is a required index field for every data type, I'm thinking maybe we could have a test to ensure that `Participant_Identifier` is in all the keys in the dict of `INDEX_FIELD_MAP`. This is more of future proofing/double checking that we don't modify code that would accidentally affect this. Thoughts?

Reply:
I don't want to assume that `ParticipantIdentifier` will be included with every data type going forward. That's a CE decision, and why I explicitly included the field in the `INDEX_FIELD_MAP` rather than specifying it once in this function.